packages/core/codegen.yml

```diff
@@ -0,0 +1,23 @@
+schema: '../../node_modules/@nft/api-graphql/schema.graphql'
+documents:
+  - './fragments/*.graphql'
+  - './mutations/*.graphql'
+generates:
+  src/graphql.ts:
+    plugins:
+      - 'typescript'
+      - 'typescript-operations'
+      - 'typescript-graphql-request'
+    config:
+      enumsAsTypes: true
+config:
+  scalars:
+    URI: 'string'
+    UUID: './types#UUID'
+    Datetime: 'string'
+    Cursor: 'string'
+    JSON: './types#EIP712Data'
```

Comment:
Not sure this one make sense.
This dedicated type should be added to the API first.

Reply:
Currently, we only use json for eip712, but indeed it should be better to have that in the API.
We can wait to have that at the API first before taking doing that 👍

Reply:
Found a good way to have this typesafe with `unknown` so we still have to cast with the `as ...` but at least now it's forced